cmd.go

```diff
@@ -137,59 +139,28 @@ func (r *Runner) preRunE(cmd *cobra.Command, args []string) error {
 	return r.prevPreRunE(cmd, args)
 }
 
-func (r *Runner) runE(runner interface{}, cfg interface{}) func(cmd *cobra.Command, args []string) error {
+func (r *Runner) runE(module fx.Option) func(cmd *cobra.Command, args []string) error {
 	return func(cmd *cobra.Command, args []string) error {
-		// Unmarshal the configuration
-		if err := Unmarshal(r.prefix, cfg); err != nil {
-			return errors.Wrap(err, "unable to unmarshal configuration")
-		}
-
```

Comment:
For backwards compatibility, I think we'll need to add this back for now (maybe mark to be removed at some point in the future.